Scripts/DCS-BIOS/lib/modules/aircraft_modules/F-15E.lua

```diff
@@ -0,0 +1,1315 @@
+F_15:defineIndicatorLight("R_MC_CAN_UNLOCK_L", 1173, "Rear Main Caution Lights Panel", "REAR CANOPY UNLOCKED Light (red)")
+F_15:defineIndicatorLight("R_MC_LOW_ALT_L", 1174, "Rear Main Caution Lights Panel", "REAR LOW ALT Light (red)")
+F_15:defineIndicatorLight("R_MC_MASTER_CAUTION_L", 1177, "Rear Main Caution Lights Panel", "REAR MASTER CAUTION Light (yellow)")
+F_15:defineIndicatorLight("R_MC_TF_FAIL_L", 1178, "Rear Main Caution Lights Panel", "REAR TF FAIL Light (red)")
+F_15:defineIndicatorLight("R_MC_ENG_L", 1179, "Rear Main Caution Lights Panel", "REAR ENGINE Light (yellow)")
+F_15:defineIndicatorLight("R_MC_HYD_L", 1180, "Rear Main Caution Lights Panel", "REAR HYD Light (yellow)")
+F_15:defineIndicatorLight("R_MC_FLT_CONT_L", 1181, "Rear Main Caution Lights Panel", "REAR FLT CONTROL Light (yellow)")
+F_15:defineIndicatorLight("R_MC_AV_BIT_L", 1182, "Rear Main Caution Lights Panel", "REAR AV BIT Light (yellow)")
+F_15:defineIndicatorLight("R_MC_MASTER_ARM_L", 1183, "Rear Main Caution Lights Panel", "REAR MASTER ARM Light (green)")
+F_15:defineIndicatorLight("R_MC_AP_L", 1184, "Rear Main Caution Lights Panel", "REAR A/P Light (green)")
+F_15:defineIndicatorLight("R_MC_PROGRAM_L", 1185, "Rear Main Caution Lights Panel", "REAR PROGRAM Light (green)")
+F_15:defineIndicatorLight("R_MC_MIN_L", 1186, "Rear Main Caution Lights Panel", "REAR MINIMUM Light (yellow)")
+F_15:defineIndicatorLight("R_MC_DSPFLOLO_L", 1187, "Rear Main Caution Lights Panel", "REAR DISPLAY FLOW LOW Light (yellow)")
+F_15:defineIndicatorLight("R_MC_AI_L", 1188, "Rear Main Caution Lights Panel", "REAR AI Light (red)")
+F_15:defineIndicatorLight("R_MC_SAM_L", 1189, "Rear Main Caution Lights Panel", "REAR SAM Light (red)")
+F_15:defineIndicatorLight("R_MC_OBST_L", 1190, "Rear Main Caution Lights Panel", "REAR OBST Light (red)")
+F_15:defineIndicatorLight("R_MC_L_GEN_L", 1191, "Rear Main Caution Lights Panel", "REAR L GEN Light (yellow)")
+F_15:defineIndicatorLight("R_MC_R_GEN_L", 1192, "Rear Main Caution Lights Panel", "REAR R GEN Light (yellow)")
+F_15:defineIndicatorLight("R_MC_EMIS_L", 1193, "Rear Main Caution Lights Panel", "REAR EMIS LMT Light (yellow)")
+F_15:defineIndicatorLight("R_MC_FUEL_L_L", 1194, "Rear Main Caution Lights Panel", "REAR FUEL LOW Light (yellow)")
+F_15:defineIndicatorLight("R_MC_NUC_L", 1195, "Rear Main Caution Lights Panel", "REAR NUCLEAR Light (yellow)")
+F_15:defineIndicatorLight("R_MC_UNARM_ATF_L", 1196, "Rear Main Caution Lights Panel", "REAR UNARMED NO ATF Light (yellow)")
+F_15:defineIndicatorLight("R_MC_CHAFF_L", 1197, "Rear Main Caution Lights Panel", "REAR CHAFF Light (yellow)")
+F_15:defineIndicatorLight("R_MC_FLARE_L", 1198, "Rear Main Caution Lights Panel", "REAR FLARE Light (yellow)")
+F_15:defineIndicatorLight("R_MC_OXY_L", 1199, "Rear Main Caution Lights Panel", "REAR OXYGEN Light (yellow)")
+
+--Externals
+F_15:defineIntegerFromGetter("EXT_SPEED_BRAKE", function()
```

Comment:
Remember to swap these out with defineFloatFromDrawArgument and defineBitFromDrawArgument

Reply:
done